### PR TITLE
Update Copilot docs

### PR DESCRIPTION
**Pull request by ntrogh**

### CODE REVIEW

File: docs/copilot/ai-powered-suggestions.md [hunk 1]
```diff
@@ -1,208 +1,93 @@
...
-
-<iframe width="560" height="315" src="https://www.youtube.com/embed/Fi3AJZZregI" title="Get Started with the Future of Coding: GitHub Copilot" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+GitHub Copilot provides autocomplete-style code completions from an AI pair programmer as you code. As you start writing code or code-related items (comments, test, and more), Copilot presents suggestions automatically in the editor to help you code more efficiently. GitHub Copilot provides suggestions for numerous languages and a wide variety of frameworks.
 
 ## Getting started
 
-If you want to use GitHub Copilot, you either need a subscription for GitHub Copilot in your personal account, or you need to be assigned a seat by your organization.
-
-The following tutorials walk you through using GitHub Copilot in VS Code.
+> **Note**: After reviewing this topic, you can get started with the introductory [Copilot getting started tutorial](/docs/copilot/getting-started-chat.md) to get set up and take a hands-on tour of Copilot in VS Code.
```
Comment (bamurtaugh) [inline docs/copilot/ai-powered-suggestions.md:16]: nit: should this state it's a hands-on tour of Copilot Chat?

```suggestion
> **Note**: After reviewing this topic, you can get started with the introductory [Copilot getting started tutorial](/docs/copilot/getting-started-chat.md) to get set up and take a hands-on tour of Copilot Chat in VS Code.
```

**Review by bamurtaugh — COMMENTED** [review]
These updates look great, thanks so much for working on them! Left a few comments.

File: docs/copilot/faq.md [hunk 2]
```diff
@@ -11,23 +11,44 @@ MetaDescription: Frequently asked questions for using GitHub Copilot in Visual S
...
 
+## GitHub Copilot subscription
+
+### My Copilot subscription is not detected in VS Code
+
+- To use GitHub Copilot Chat in Visual Studio Code, you must be signed into Visual Studio Code with the same GitHub ID that has access to GitHub Copilot. If your Copilot subscription is associated with another GitHub account, you might have to sign out of your GitHub account and sign in with another account. Use the **Accounts** menu in the Activity bar for signing out of your current GitHub account.
+
+- Verify that your Copilot subscription is still active in [GitHub Copilot settings](https://github.com/settings/copilot).
+
+### How can I get a Copilot subscription?
```
[the remark this answers is not in the document]
Comment (ntrogh) [reply docs/copilot/faq.md:22]: Good one. It should.

File: docs/copilot/faq.md [hunk 3]
```diff
@@ -47,10 +68,23 @@ You can tell if you're running a pre-release version by the "Pre-release" badge
...
 Check each requirement if Copilot Chat doesn't work:
 
-* Make sure you are on the latest version of Visual Studio Code (run **Code: Check for Updates**).
-* Make sure you have the latest version of both the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) and [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) extensions.
-* Your GitHub account that is signed into VS Code must have an activated Copilot subscription. Check your [Copilot subscription](https://github.com/settings/copilot).
+- Make sure you are on the latest version of Visual Studio Code (run **Code: Check for Updates**).
+- Make sure you have the latest version of both the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) and [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) extensions.
+- Your GitHub account that is signed into VS Code must have an active Copilot subscription. Check your [Copilot subscription](https://github.com/settings/copilot).
+
+## Troubleshooting
```
[the remark this answers is not in the document]
Comment (ntrogh) [reply docs/copilot/faq.md:79]: I've put them after each other. I also restructure the top-level sections to separate out "Code completions" from "General".

File: docs/copilot/getting-started.md [hunk 4]
```diff
@@ -1,134 +1,183 @@
...
-    ![Video showing Copilot suggesting a code fix](./images/getting-started/copilot-code-fix.gif)
+    Directly from the chat response, you can optionally select the **Insert into Terminal** button to copy the proposed command in your terminal.
 
 ## Congratulations
 
-Congratulations, you've now used artificial intelligence to enhance your coding! In this tutorial, you installed the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) extension in VS Code and used the Copilot to provide you with suggestions in the editor to help you code more efficiently.
+Congratulations, you've now used artificial intelligence to enhance your coding! In this tutorial, you successfully set up Copilot in VS Code, and used Copilot code completions, Copilot Chat, and code actions to help you code more efficiently.
 
-Next, learn how you can [get started with Copilot chat features](/docs/copilot/getting-started-chat.md) to interact with Copilot and ask questions about your code, your workspace, and more.
+Next, learn more about GitHub Copilot Chat in the [Copilot Chat Tutorial](/docs/copilot/getting-started-chat.md).
```
[the remark this answers is not in the document]
Comment (ntrogh) [reply docs/copilot/getting-started.md:183]: Rephrased it a bit to ease the transition into the Chat tutorial.

File: docs/copilot/ai-powered-suggestions.md (hunk 1 above)
Comment (ntrogh) [reply docs/copilot/ai-powered-suggestions.md:16]: The link was incorrect. It should actually point to the Getting started tutorial, not the Chat one. People on this article should have as a next step to get started with the intro tour.

**Comment by ntrogh** [comment]
@bamurtaugh all articles are now ready for review.

File: docs/copilot/overview.md [hunk 5]
```diff
@@ -21,162 +21,101 @@ With GitHub Copilot in VS Code you can:
...
-In addition to inline suggestions, you can also get assistance from Copilot via a chat interface. You can use Copilot Chat in VS Code in different ways, depending on the context:
+Follow these steps to [Set up GitHub Copilot in VS Code](/docs/copilot/getting-started.md#set-up-vs-code-for-github-copilot).
 
-* **Chat view:** Ask Copilot for help with any task or question in the Chat view, bringing in code from your current files.
-* **Inline Chat:** Apply Copilot's AI suggestions directly to your code, staying in the flow.
-* **Quick Chat:** Bring up a Chat-powered dropdown for quick questions and suggestions.
+## Copilot tutorials
 
-To use the chat features in VS Code, install the [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) extension.
+* [GitHub Copilot quickstart](/docs/copilot/getting-started.md)
```
Comment (bamurtaugh) [inline docs/copilot/overview.md:36]: Should this mention completions specifically? As a new reader, I might wonder what the difference is between this one and the next tutorial at first glance of these titles.

File: docs/copilot/overview.md [hunk 6]
```diff
@@ -21,162 +21,101 @@ With GitHub Copilot in VS Code you can:
...
+* [Get started with Copilot Chat](/docs/copilot/getting-started-chat.md)
 
-> <a class="install-extension-btn" href="vscode:extension/GitHub.copilot-chat">Install the GitHub Copilot Chat extension</a>
+## Use cases for GitHub Copilot in VS Code
```
Comment (bamurtaugh) [inline docs/copilot/overview.md:39]: I absolutely love this new use cases section! For even easier navigation and scanning, I wonder if a quick bulleted list that linked to the top-level categories (i.e. Code completions in the editor, Answering coding questions) would help. This way, I can quickly scan what kinds of scenarios Copilot helps with and potentially click to those I find most interesting first, without needing to scroll.

Comment (bamurtaugh) [reply docs/copilot/overview.md:39]: I also think either videos or screenshots would work well. If @burkeholland or others have a video for a specific topic, that could be really cool to integrate, but given the number of scenarios we'd like to cover here, I think screenshots are fast and effective.

File: docs/copilot/getting-started-chat.md [hunk 7]
```diff
@@ -15,140 +15,182 @@ If you're new to using GitHub Copilot in VS Code, see the [GitHub Copilot Gettin
...
 
-As you're iterating over your code, there might be moments where you need help from your AI pair programmer to accomplish a task. In the [Copilot Getting Started](/docs/copilot/getting-started.md), you used Copilot inline suggestions and optionally provided context by using code comments.
+1. Open the Chat view from the Activity Bar or press `kb(workbench.action.chat.open)`.
 
-In some cases, it might be more efficient to just ask Copilot about a specific topic by using a chat message. For example, a common cause for errors is when a method receives invalid input. In this step, you start with a simple calculator in TypeScript, and use GitHub Copilot Chat to refactor the code to add error handling.
+    ![Screenshot of VS Code editor, showing the Copilot Chat view, highlighting the chat control in the Activity bar.](./images/getting-started-chat/copilot-chat-view.png)
 
-![Video that shows asking Copilot to add error handling to our code. Copilot presents a diff view with the suggested code changes.](./images/getting-started-chat/copilot-chat-add-error-handling.gif)
+    > **Tip**: Drag the Chat view to the Secondary Side bar to keep both the Chat view and Explorer view open.
```
Comment (bamurtaugh) [inline docs/copilot/getting-started-chat.md:34]: I love this tip (and frequently give it in my sessions), though readers might not know what the secondary sidebar is (and I wonder if they could be confused why doesn't the above image show it). Maybe just linking to a doc on what the secondary sidebar is could help?

File: docs/copilot/getting-started-chat.md [hunk 8]
```diff
@@ -15,140 +15,182 @@ If you're new to using GitHub Copilot in VS Code, see the [GitHub Copilot Gettin
...
 
-Let's use GitHub Copilot to help us add error handling to our code.
+1. In the chat input field, enter *nodejs web server frameworks*, and then press `kb(inlineChat.accept)` to send the request to Copilot.
 
-1. Open Visual Studio Code and create a new TypeScript file `Calculator.ts`.
+    Copilot returns a list of web frameworks and with some additional information. So, without leaving VS Code, you can search for coding information while you stay in VS Code. Note that your results might be different from what is shown in the screenshots.
```
Comment (bamurtaugh) [inline docs/copilot/getting-started-chat.md:38]: Should we give a bit of context about why your results might be a bit different (i.e. AI is non-deterministic)?

File: docs/copilot/prompt-crafting.md [hunk 9]
```diff
@@ -25,7 +25,7 @@ Copilot works best when it has sufficient context to know what you're doing and
 
 #### Open files
```
Comment (bamurtaugh) [inline docs/copilot/prompt-crafting.md:26]: I initially missed the fact these were grouped into 2 paths (inline vs chat tips), so I had left a couple comments about "Should we divide these into 2 paths?" 😄 

But considering I missed the divide, I think readers could too. Maybe a quick note ~line 16 (so before getting into the inline path) could be helpful, something just noting that there are different ways to prompt craft for completions vs chat, and this doc has different tips for both. Maybe also linking to the headers (inline vs chat) so I don't have to scroll or ctrl+f to get to the chat tips if those are what I want.

**Review by bamurtaugh — COMMENTED** [review]
These changes look fantastic! Added a few comments, lmk if I can help explain anything.

File: docs/copilot/overview.md (hunk 6 above)
Comment (ntrogh) [reply docs/copilot/overview.md:39]: Agree. It might be too distracting, I'll incorporate more videos in the individual articles.

**Pull request merged**